packages/terra-clinical-result/src/proptypes/valuePropTypes.js

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
+
+const valueQuantityPropShape = PropTypes.shape({
+  /**
+   *  Numeric Value for an Observation Result
+   */
+  value: PropTypes.number,
+  /**
+   *  Unit of Measure representation for an Observation Result
+   */
+  unit: PropTypes.string,
+  /**
+   *  Optional (FHIR) - System that defines coded unit form
+   */
+  system: PropTypes.string,
```

Comment:
From what I can tell,`system` and `code` are never used. Lets remove them.

Reply:
Intent was to match the FHIR spec ([Observation](https://www.hl7.org/fhir/observation.html) > specifically [Value Quantity](https://www.hl7.org/fhir/datatypes.html#Quantity)), though we don't do anything with it yet, not sure if it applies to other views or should later in other clinical-result components. We could remove and make sure to provide a test and example with FHIR data to make sure it doesn't have any effect.